jprov/utils/network.go

```diff
@@ -165,17 +165,20 @@ func WriteFileToDisk(cmd *cobra.Command, reader io.Reader, file io.ReaderAt, clo
 	tree = nil // for GC
 
 	f, err := os.OpenFile(GetStoragePathForTree(clientCtx, fid), os.O_WRONLY|os.O_CREATE, 0o666)
+	defer func() {
+		if tmpErr := f.Close(); tmpErr != nil && err == nil {
+			err = tmpErr
```

Comment:
where does this error get handled?

Reply:
Thank you for catching that, I found a better solution for this.